datasetinsights/commands/download.py

```diff
@@ -79,52 +73,14 @@ def convert(self, value, param, ctx):
         "binary files."
     ),
 )
-@click.option(
```

Comment:
excellent! I just the term `access token` is used in this [doc](https://github.com/Unity-Technologies/Unity-Simulation-Docs/blob/master/doc/cli.md#usim-inspect-auth). We should stick with this name for the whole repo!

Reply:
What do you mean by "just"

Reply:
typo. `I just realized the term....` :